contracts/mocks/Cover/CoverMockStakingPool.sol

```diff
@@ -92,6 +93,10 @@ contract CoverMockStakingPool is IStakingPool, ERC721Mock {
   ) external {
   }
 
+  function setProducts(ProductParams[] memory params) external {
```

Comment:
Are the  addProducts, removeProducts, editProducts methods deferred for another PR or we're scraping them entirely?

Reply:
the idea is for the `setProducts` function to implement all possible operations and have a unique function instead of the 3 you enumerated